resources/views/galerieLayout.blade.php

```diff
@@ -8,5 +8,50 @@
 
 </div>
 <button class="afficher" type="button" class="titre" onclick="voir()">Tout Voir</button>
+<!-- titres de contact -->
+<div class="contacttitre">
+  <h2> Contactez-moi en remplissant le formulaire ci-dessous :</h2>
+</div>
+
+<!-- Formulaire -->
+<div class="formulaire_presta">
+    <form action="/contact" role="form" method="POST" id="reused_form">
```

Comment:
WTF ???
Where is the code which sends the message ?
A specific action would be better…